src/tests/ftest/nvme/pool_exclude.py

```diff
@@ -59,8 +59,8 @@ def run_nvme_pool_exclude(self, num_pool, oclass=None):
             oclass = self.ior_cmd.dfs_oclass.value
 
         # Exclude rank :  ranks other than rank 0.
-        exclude_servers = len(self.hostlist_servers) * 2
-        rank_list = list(range(1, exclude_servers))
+        rank_list = list(self.server_managers[0].ranks.keys())
+        rank_list.remove(0)
```

Comment:
FYI that special handling for rank 0 is starting to be less of a guarantee. E.g. it isn't guaranteed that rank 0 is the pool leader